test/transport/IceIntegrationTest.cpp

```diff
@@ -293,7 +293,7 @@ TEST_F(IceIntegrationTest, dtlsRace)
     }
 
     bool connected = false;
-    while (!connected && utils::Time::getAbsoluteTime() - startTime < 5 * utils::Time::sec)
+    while (!connected && utils::Time::getAbsoluteTime() - startTime < 20 * utils::Time::sec)
```

Comment:
will revert before final commit

Reply:
It will exit as soon as they are connected. Giving them a bit more time since there are many connections.